recipes/medcoupling/bld.bat

```diff
@@ -0,0 +1,41 @@
+mkdir build
+cd build
+
+:: -fdefault-integer-8 is not supported on msvc. Should consider moving to clang/msys2
+set FCFLAGS=-fdefault-integer-8 %FCFLAGS%
+set FFLAGS=-fdefault-integer-8 %FFLAGS%
+set CMAKE_CXX_FLAGS=-fvisibility=hidden %CMAKE_CXX_FLAGS%
+
+cmake -G "Ninja" %SRC_DIR% ^
+    -Wno-dev ^
+    -D CMAKE_BUILD_TYPE=%BUILD_TYPE% ^
+    -D PYTHON_ROOT_DIR="%PREFIX%" ^
+    -D PYTHON_EXECUTABLE:FILEPATH="%PYTHON%" ^
+    -D CONFIGURATION_ROOT_DIR="%SRC_DIR%/deps/config" ^
+    -D SALOME_CMAKE_DEBUG=ON ^
+    -D SALOME_USE_MPI=OFF ^
+    -D MEDCOUPLING_BUILD_TESTS=OFF ^
+    -D MEDCOUPLING_BUILD_DOC=OFF ^
+    -D MEDCOUPLING_USE_64BIT_IDS=ON ^
+    -D MEDCOUPLING_USE_MPI=OFF ^
+    -D MEDCOUPLING_MEDLOADER_USE_XDR=OFF ^
+    -D MEDCOUPLING_INSTALL_PYTHON=%SP_DIR% ^
+    -D XDR_INCLUDE_DIRS="" ^
+    -D MEDCOUPLING_PARTITIONER_PARMETIS=OFF ^
+    -D MEDCOUPLING_PARTITIONER_METIS=OFF ^
+    -D MEDCOUPLING_PARTITIONER_SCOTCH=OFF ^
+    -D MEDCOUPLING_PARTITIONER_PTSCOTCH=OFF ^
+    -D HDF5_ROOT_DIR="%LIBRARY_PREFIX%" ^
+    -D SWIG_ROOT_DIR="%LIBRARY_PREFIX%" ^
+    -D MEDFILE_ROOT_DIR="%LIBRARY_PREFIX%" ^
+    -D CMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
+    %CMAKE_ARGS%
+
+if errorlevel 1 exit 1
+ninja
+if errorlevel 1 exit 1
+ninja install
+
+:: Generate pyi stubs for binary module
+%PYTHON% %RECIPE_DIR%/stubs/custom_stubs_gen.py
```

Comment:
Why does this get called for windows but not unix.

Reply:
oh, then that's a mistake on my part. The stub generator isn't quite finished yet. I'll remove it and the related dependencies for now and return to it once I've made more progress on it

Reply:
Honestly, this is the kind of feature that "seems nice for all" but I am not sure it should be added until attempts to reach out to upstream are.

There is a slippery slope between "adding a nice feature" and "forking" a package.

If that is ok, please remove the stub generator at this stage, and make attempts to reach out to upstream and contribute it.

Otherwise, maybe live without type hints for this package?